packages/components/src/internal/components/auditlog/AuditQueriesListingPage.tsx

```diff
@@ -28,8 +28,6 @@ import { AuditDetailsModel } from './models';
 import { getAuditDetail } from './actions';
 import { AuditQuery, getAuditQueries } from './utils';
 
-const AUDIT_QUERIES = getAuditQueries();
```

Comment:
Note: I believe @labkey-ankurj has a very similar (or the same) change in his open PR for ui-components.

Reply:
https://github.com/LabKey/labkey-ui-components/pull/346/files

Reply:
Thanks for heads up.  I talked with Ankur and we're going to merge in my branch and close that one.  I've got his changes in here